### PR TITLE
Cleanup unused generated code

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 1)

**Review by CyrusNajmabadi — APPROVED** [review]

File: src/Compilers/CSharp/Portable/Generated/CSharpSyntaxGenerator/CSharpSyntaxGenerator.SourceGenerator/Syntax.xml.Internal.Generated.cs [hunk 1]
```diff
@@ -45692,248 +45692,5 @@ public static NullableDirectiveTriviaSyntax NullableDirectiveTrivia(SyntaxToken
 
             return new NullableDirectiveTriviaSyntax(SyntaxKind.NullableDirectiveTrivia, hashToken, nullableKeyword, settingToken, targetToken, endOfDirectiveToken, isActive);
         }
-
-        internal static IEnumerable<Type> GetNodeTypes()
```
Comment (cston) [inline src/Compilers/CSharp/Portable/Generated/CSharpSyntaxGenerator/CSharpSyntaxGenerator.SourceGenerator/Syntax.xml.Internal.Generated.cs:45696]: >GetNodeTypes [](http://example.com/codeflow?start=42&length=12)

Can we remove the corresponding method from `Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax.SyntaxFactory` as well?

File: .vscode/tasks.json [hunk 2]
```diff
@@ -98,7 +98,7 @@
     },
     {
         "label": "generate compiler code",
-        "command": "${workspaceFolder}/.dotnet/dotnet",
```
Comment (Youssef1313) [inline .vscode/tasks.json:101]: `.dotnet` directory doesn't have to exist locally. So, running the task fails. This fixed it for me.

Comment (cston) [reply .vscode/tasks.json:101]: I'm not sure what the implications of this change are, and it doesn't seem directly related to the generator change. Also, there are other uses of the same `"command"` in this file.

Let's avoid changing this file in this PR.

File: .vscode/tasks.json [hunk 3]
```diff
@@ -98,7 +98,7 @@
...
+        "command": "dotnet",
```
Comment (Youssef1313) [inline .vscode/tasks.json:101]:
```suggestion
        "command": "${workspaceFolder}/.dotnet/dotnet",
```

**Review by cston — APPROVED** [review]

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 3)

**Pull request merged**